jenkinsfiles/ginkgo-gke.Jenkinsfile

```diff
@@ -199,7 +182,6 @@ pipeline {
   }
     post {
         always {
-            sh 'cd ${TESTDIR}/gke; ./stop-registry.sh || true'
```

Comment:
Same here, it looks like `stop-registry.sh` is now no longer used. Remove it as well?